src/CLIUtils.ts

```diff
@@ -112,9 +180,19 @@ export async function runFiles(
       opts.log && console.log(`Finished running ${file}`);
     } catch (err) {
       opts.log && console.error(`Error running ${file}`, err);
-      throw err;
```

Comment:
This change changes the behavior of the CLI:
* Before: when a replay is failed, we were throwing error from here so that the `yargs` library was returning the status code `1` to the bash
* With this change: even though a replay is failed, we now return control to the user with status code `0`.

We want to return status code `1` when a replay is failed since this makes the library more useful especially in CI workflows (i.e. if a replay is failed, CI will report the task to be failed as well)

So, can you restore the old behavior of throwing an error? (It's fine running all the results, I guess at the end of all runs, we can check whether there is a replay result with `failure` and if so throw an error -- though up to you 😂)